server/team/team.route.js

```diff
@@ -8,9 +8,17 @@ const router = express.Router(); // eslint-disable-line new-cap
 router
   .route('/')
   /** GET /api/teams - Get list of users */
-  // .get(teamCtrl.list)
-
+  .get(teamCtrl.list)
   /** POST /api/teams - Create new team */
   .post(ifNoUserRedirect(), teamCtrl.create);
 
+router
+  .route('/analyze')
+  .get(teamCtrl.analyze);
```

Comment:
I could maybe see a use for this route for when a team member want to trigger another test of their site manually without the webhook. It would need some middleware for protection, like `ifNoUserRedirect` and a new one like `ifUserHasNoTeamRedirect`.

We would also need to add a check to see if a test is scheduled or pending, so that we don't end up queuing 50 tests for one version of the site

This route should not be able to test 300+ sites any time someone hits it.